web-common/src/features/dashboards/state-managers/selectors/dashboard-queries.ts

```diff
@@ -1,181 +1,17 @@
-import { mergeMeasureFilters } from "@rilldata/web-common/features/dashboards/filters/measure-filters/measure-filter-utils";
```

Comment:
Nice seeing this massive removal! This was needed when we had different APIs used. Now with a single AggregationRequest I was thinking of writing up a wrapper around it using builder pattern. Might decrease the code a lot more.